aria/logger.py

```diff
@@ -152,15 +152,14 @@ def __init__(self, session, engine, **kwargs):
         # Cyclic dependency
         from aria.storage.modeling.model import Log
         self._cls = Log
+        # This is needed since the engine and session are entirely new
+        self._cls.__table__.create(bind=self._engine, checkfirst=True)
```

Comment:
check if any self._engine syntax exists (at the very least move to the instantiation func)